mirrord-agent/src/runtime.rs

```diff
@@ -1,68 +1,70 @@
 use std::{
     fs::File,
     os::unix::io::{IntoRawFd, RawFd},
+    path::PathBuf,
 };
 
 use anyhow::{anyhow, Result};
+use bollard::{container::InspectContainerOptions, Docker};
 use containerd_client::{
     connect,
-    services::v1::{containers_client::ContainersClient, GetContainerRequest},
+    services::v1::{tasks_client::TasksClient, GetRequest},
     tonic::Request,
     with_namespace,
 };
 use nix::sched::setns;
-use serde::{Deserialize, Serialize};
+use tracing::debug;
 
 const CONTAINERD_SOCK_PATH: &str = "/run/containerd/containerd.sock";
 const DEFAULT_CONTAINERD_NAMESPACE: &str = "k8s.io";
+pub(crate) enum Runtime {}
```

Comment:
I don't see any reason this would be an Enum with current design (totally okay to have just public functions..)